index.js

```diff
@@ -17,24 +17,26 @@ module.exports = function(filteredImports) {
 
       ExpressionStatement: {
         enter: function() {
-          shouldRemove.push(false);
+          if (shouldRemove) {
```

Comment:
@rwjblue this is how the visitors are called for the `default` fixture:

```
Program enter
Identifier a
ExpressionStatement enter
Identifier a
Identifier ok
ExpressionStatement exit
Program exit
Identifier _interopRequireDefault
Identifier obj
Identifier obj
Identifier obj
Identifier __esModule
Identifier obj
Identifier default
Identifier obj
Identifier _assert
Identifier require
Identifier _assert2
Identifier _interopRequireDefault
Identifier _assert
```

so basically it's traversing the source file, then applies the modules transformation and traverses the file again without wrapping it in a `Program` node

I've managed to slim it down though by initializing the `shouldRemove` stack much earlier and not resetting it back to `undefined` after `Program.exit`. It seems to work fine for our test fixtures and I don't see any reason why this shouldn't work in production code too.

/cc @mmun